homeassistant/components/roomba/__init__.py

```diff
@@ -68,9 +68,7 @@ def _has_all_unique_blids(value):
         return True
     for index, conf in enumerate(config[DOMAIN]):
         _LOGGER.debug("Importing Roomba #%d - %s", index, conf[CONF_HOST])
-        current_config_entry = _async_find_matching_config_entry(
-            hass, conf[CONF_PREFIX]
-        )
+        current_config_entry = _async_find_matching_config_entry(hass, conf[CONF_BLID])
```

Comment:
We don't need to update the config entries here since they can resetup the auth details if it changes.